master/usage/configuration/mtu.md

```diff
@@ -22,12 +22,13 @@ packet.
 | 9000 | 9000 | 8980 | 8950 |
 | 1460 (GCE) | 1460 | 1440 | 1410 |
 | 9001 (AWS Jumbo) | 9001 | 8981 | 8951 |
+| 1450 (OpenStack VXLAN) | 1450 | 1430 | 1400 |
 
 #### Default MTU sizes
 
 The default MTU for workload interfaces is 1500, this is to match the most
 common network MTU size.  The default MTU for the IP-in-IP tunnel device
-is 1440 to match the value needed in GCE.
+is 1430 to match the value needed in OpenStack VXLAN.
```

Comment:
Is there a PR up to change the default MTU that is configured for the IP-in-IP tunnel?  We probably don't want to update this line until that is merged.
If there is one or has already been merged could you add a link to it, I tried searching for one but couldn't find it.

Reply:
In order to get this PR merge, can we remove this diff and just include the change to the table above?

We can add this back once the felix PR has been agreed / merged.

Reply:
Done @caseydavenport